src/main/java/org/mule/modules/watsonalchemylanguage/WatsonAlchemyLanguageConnector.java

```diff
@@ -154,33 +154,25 @@ public Feeds feedDetection(@Default("#[payload]") String url) {
 				.execute();
 	}
 
+	
 	/**
 	 * Extract concepts from a webpage or plain text.
 	 *
 	 * API Doc: {@see http://www.ibm.com/watson/developercloud/alchemy-language/api/v1/#concepts}
-	 *
-	 * @param source The text or url to process
-	 * @param maxRetrieve Maximum number of entities to return (default = 50) detected entities by default)
-	 * @param knowledgeGraph Set this to 1 to include knowledge graph information in the results. This incurs an
-	 *            additional transaction charge
-	 * @param linkedData Set this to 0 to hide Linked Data content links in the response
-	 * @param showSourceText Check this to include the source text in the response TwitterHandle, Hashtag, and IPAddress
-	 * @param cquery A visual constraints query to apply to the web page. Required when sourceText is set to cquery
-	 * @param xpath An XPath query to apply to the web page. Required when sourceText is set to one of the XPath values
-	 * @param sourceText How to obtain the source text from the webpage
 	 * 
-	 * @return Return a list of detected high level concepts used in the text, webpage or HTML content.
+	 * @param request Request with all the options for a combine call operation.
```

Comment:
You need to be careful when you copy and paste, the proper description is "Request with all the options for the extract concepts operation."